.github/workflows/gke.yaml

```
@@ -160,6 +199,45 @@ jobs:
         run: |
           cilium connectivity test
 
+      - name: Clean up Cilium
+        run: |
+          cilium uninstall --wait
+          pgrep -f "kubectl port-forward" | xargs kill -9 # kill background port forwards
+
+      - name: Install Cilium With Encryption and Tunnel Datapath
+        run: |
+          cilium install \
+            --cluster-name=${{ env.clusterName }} \
```

I think we're missing something here... 😇 
```suggestion
            --encryption \
            --cluster-name=${{ env.clusterName }} \
```

---

oops good catch I've add this now.